data/db-schema/src/main/resources/net/corda/db/schema/config/migration/cpx-creation-v1.0.xml

```diff
@@ -176,6 +176,34 @@
                                  baseTableSchemaName="${schema.name}"
                                  referencedTableSchemaName="${schema.name}"/>
 
+        <createTable tableName="cpk_db_change_log_audit" schemaName="${schema.name}">
+            <column name="cpk_name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="cpk_version" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="cpk_signer_summary_hash" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="cpk_file_checksum" type="VARCHAR(255)">
```

Comment:
I'd had a tonne of thoughts about the pk. Ideally it'd include the entity version, but it doesn't seem to update properly, though I'm convinced that's an issue in the Cpi code and I don't want to start touching that too much.

cpk_file_checksum isn't a runner, as you always end up with multiple entries per checksum.
![image](https://user-images.githubusercontent.com/10749142/191513943-7d4ec07b-c532-42b7-91a5-65702cded580.png)

Reply:
You could also upload a cpi, replace it, and the reupload it and it will create the same entires again, so even if you have the checksum and the filepath, that can also reoccur.

Reply:
We may also want to re-run the exact same one from earlier, but probably still want a record of that having been done. Might need to add more info later, or another table to track which migrations are actually ran, but I'm leaving that until a little later